staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -88,8 +89,7 @@ var (
 	//
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
-	// TODO(audit): figure out a general way to let the client choose their preferred version
```

Comment:
Some suggestions about this TODO:
https://github.com/kubernetes/kubernetes/issues/48561#issuecomment-319225619